automated install/basic-install.sh

```diff
@@ -1261,6 +1267,9 @@ installConfigs() {
         chown ${LIGHTTPD_USER}:${LIGHTTPD_GROUP} /var/cache/lighttpd/compress
         mkdir -p /var/cache/lighttpd/uploads
         chown ${LIGHTTPD_USER}:${LIGHTTPD_GROUP} /var/cache/lighttpd/uploads
+
+        # Let PHP edit the regex file
+        chown pihole:${LIGHTTPD_GROUP} "${regexFile}"
```

Comment:
And another guard on this call, just in case the file is missing, then we won't get any errors leaking. Or how ever you'd like to swallow errors.

Reply:
The file should exist at this point, if we're creating it above. If this would be not run when the file already exists, it could lead to it not being editable from the web interface.

Reply:
I agree that it should exist at that point, but I'd like to have more error handling in the code as we go forward. We can add that now, or I can look at it when I go over the installer overhaul and add more error handling for the whole script. Just trying to get ahead on some things, the rest of the function will have protections added in later.